src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -103,7 +104,13 @@ public Person toModelType() throws IllegalValueException {
         final Address modelAddress = new Address(address);
 
         final Set<Tag> modelTags = new HashSet<>(personTags);
-        return new Person(modelName, modelPhone, modelEmail, modelAddress, modelTags);
+        Nok placeholderNok = new Nok(
+                new Name("Bernice Yu"),
+                new Phone("99272758"),
+                new Email("berniceyu@example.com"),
+                new Address("Blk 30 Lorong 3 Serangoon Gardens, #07-18")
+        );
+        return new Student(modelName, modelPhone, modelEmail, modelAddress, placeholderNok, modelTags);
```

Comment:
Again same question is Bernice the default NOK? does following the same format as the other fields not work?

Reply:
This is a placeholder as well, because I'm leaving the functionality for Jackson for next week. 
The same format doesn't work because nok is a class instead of a string like the other fields, so you need a nested json which isn't easy to implement